doc/Programming-guide/Key-platform-features/Data-collection/Kafka-log-appender/index.md

```diff
@@ -151,21 +93,82 @@ Example result:
 }
 ```
 
-# Playing with Kafka log appender
+## Configure log appender
+
+The Cassandra log appender configuration must match [this Avro schema]({{github_url}}server/appenders/kafka-appender/src/main/avro/kafka-appender-config.avsc).
+
+You can configure the following log appender settings:
+
+|Name                   |Description                                                                                                            |
+|-----------------------|-----------------------------------------------------------------------------------------------------------------------|
+|`bufferMemorySize`     |Message buffer size in bytes.                                                                                           |
+|`executorThreadPoolSize`|Number of threads that can simultaneously perform an operation with your Kafka log appender.                                            |
+|`kafkaAcknowledgement`   |Number of acknowledgments that the producer requires the leader to receive before considering a request complete.  |
+|`kafkaCompression`       |Type of built-in message compression.                                                                             |
+|`kafkaKeyType`           |Type of generated message key.                                                                                          |
+|`kafkaServers`           |List of Kafka bootstrap servers (host name and port pairs).                                                              |
+|`partitionCount`         |Count of event partitions.                                                                                             |
+|`retries`                |Failover property. Number of retries for message delivery failure due to no connection.                                             |
+|`topic`                  |Logs destination topic.                                                                                                 |
+|`useDefaultPartitioner`  |If false, the log appender will calculate partitions independently.                                                              |
+
+Below is an example configuration that matches the mentioned Avro schema.
+
+```json
+{
```

Comment:
Couldn't upload this example configuration in Administration UI: _Unexpected service error occurred: Expected boolean. Got START_OBJECT_
The next modified configuration uploaded successfully:
```
{
    "kafkaServers":[
        {
            "host":"localhost",
            "port":9092
        }
    ],
    "topic":"kaa",
    "useDefaultPartitioner":true,
    "partitionCount":1,
    "kafkaKeyType":{
        "org.kaaproject.kaa.server.appenders.kafka.config.gen.KafkaKeyType":"NULL"
    },
    "executorThreadPoolSize":1,
    "bufferMemorySize":33554432,
    "kafkaCompression":{
        "org.kaaproject.kaa.server.appenders.kafka.config.gen.KafkaCompression":"NONE"
    },
    "kafkaAcknowledgement":{
        "org.kaaproject.kaa.server.appenders.kafka.config.gen.KafkaAcknowledgement":"ONE"
    },
    "retries":0
}
```